packages/wrangler/src/generate/index.ts

```diff
@@ -36,33 +37,14 @@ export function generateOptions(yargs: CommonYargsArgv) {
 }
 type GenerateArgs = StrictYargsOptionsToInterface<typeof generateOptions>;
 
-// Originally, generate was a rust function: https://github.com/cloudflare/wrangler-legacy/blob/master/src/cli/mod.rs#L106-L123
 export async function generateHandler(args: GenerateArgs) {
-	// Questions
-	// Can we kick you over to C3 directly rather than go to init and then to C3? Simplify it
-	// If one passes a template name to wrangler generate myproject template-name — then we should show a warning that
-
 	logger.warn(
 		`Deprecation: \`wrangler generate\` is deprecated.\n` +
-			`Running \`npm create cloudflare@latest\` for you instead.\n\n` +
+			`Running \`npm create cloudflare@latest\` for you instead.\n` +
+			`Any arguments passed to \`wrangler generate\` will be ignored.\n\n` +
```

Comment:
Why not call C3 with the template that the user specifies and let it do it's flow?